examples/CIFAR/README.md

```diff
@@ -0,0 +1,36 @@
+The jupyter notebook is good for just trying out CIFAR using pymarlin.
+Run in [Collab](https://colab.research.google.com/drive/1kmwUenRF8GFmS4F9wg2DT7yEAE4uD5Am?usp=sharing)
```

Comment:
this can actually switch to the ipynb that exists in github with the following link (once merged to main)

https://colab.research.google.com/github/microsoft/PyMarlin/blob/main/examples/CIFAR/CIFAR.ipynb

or try with your current branch:

https://colab.research.google.com/github/microsoft/PyMarlin/blob/krkusuk/examples/examples/CIFAR/CIFAR.ipynb